### PR TITLE
Added a first relative orbits script file, into contrib

### PR DESCRIPTION
**Pull request by sammmlow**
I have created a relative orbits class (Class RelativeOrb), that takes in two poliastro.twobody.Orbit objects as input. A skeleton of the class structure is shown in the image below. The full script is in file `relative.py` in the contrib folder.

image

An example implementation of how one could use this class:

```
# Initialize an example Satellite A as the chief spacecraft.
satC = Orbit.from_classical( attractor = Earth,
                             a         = 6918.140 * u.km,
                             ecc       = 1e-6     * u.one,
                             inc       = 10.0     * u.deg,
                             raan      = 70.0     * u.deg,
                             argp      = 90.0     * u.deg,
                             nu        = 1.65     * u.deg)

# Initialize an example Satellite B as the deputy spacecraft.
satD = Orbit.from_classical( attractor = Earth,
                             a         = 6918.140 * u.km,
                             ecc       = 0.012    * u.one,
                             inc       = 11.4     * u.deg,
                             raan      = 72.35    * u.deg,
                             argp      = 135.0    * u.deg,
                             nu        = -46.5725 * u.deg)

# Instantiate the relative orbits object.
relativeSat = RelativeOrb( satC, satD )

# Propagate the relative orbit
relativeSat.propagate()

# Plot the relative trajectory in local Hill Frame (VVLH) of the chief.
relativeSat.plot()
```

image

An example relative orbit with non-zero eccentricity and non-zero inclination separations should appear as like above. Check out what happens if they have different semi-major axes! :)

### CODE REVIEW

**Review by sammmlow — COMMENTED** [review]
I have re-editted the code to make the comments and docstrings clearer :)

I have also tested it against STK on the two-body propagator and checked that the relative vectors are correctly computed! 👍 

You can scroll down to Lines 580 and more if you would like to check out how to call the relative orbits object. Code from lines 0 - 579 are mostly just the building of the relative orbits class. Looking forward to my first poliastro code review <3

**Comment by astrojuanlu** [comment]
Thanks a ton for this pull request @sammmlow ! Those plots look awesome 😍 Will review this today

**Comment by sammmlow** [comment]
Thanks! I editted some comments and docstrings I think so I re-committed :)

**Review by astrojuanlu — COMMENTED** [review]
I did a first quick pass without focusing much on the math or the style, but this looks really great!

File: contrib/relative.py [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+                             nu        = -46.5725 * u.deg)
+
+# Instantiate the relative orbits object.
+relativeSat = RelativeOrb( satC, satD )
+
+# Propagate the relative orbit
+relativeSat.propagate()
+
+# Plot the relative trajectory in local Hill Frame (VVLH) of the chief.
+relativeSat.plot()
```
[the remark this answers is not in the document]
Comment (sammmlow) [reply contrib/relative.py]: I've got to admit I didn't think about that!

Thank you for the wonderful suggestion Juan! Re-committed to the pull request = )

[the remark this answers is not in the document]
Comment (astrojuanlu) [reply contrib/relative.py]: Yeah, matplotlib is totally fine! :) Also bear in mind that we do support it, using the `poliastro.plotting.static.StaticOrbitPlotter`

**Review by astrojuanlu — APPROVED** [review]
Tested locally, this is ready to go 🚀 Thanks a lot @sammmlow and congratulations for your first contribution!

**Comment by sammmlow** [comment]
Awesome! Is it standard practice for me to delete this branch after it has been merged? :)

**Pull request merged**